src/format/opamFile.mli

```diff
@@ -92,6 +92,10 @@ module Config: sig
 
   val dl_tool: t -> arg list option
 
+ (* Field "backup-urls": a list of urls from which it is possible
+    to download by checksum, i.e. $URL/9/a/b/9ab48749e8201... *)
```

Comment:
The service is not tied to md5 checksums, it can store any checksum (md5, sha1, etc.). It is restricted to md5 here just because they are the only checksums used by OPAM.